ClinGen/clincoded
# #2082 Dashboard history timestamp fix for delete record

This PR fixes issue #2082 (incorrect dashboard delete action timestamp); the delete timestamp displayed now pins to the delete time instead of current system time.

In case anyone else wants to test out the demo site, you can access at: 
https://2082-sc-dashboard-history-timestamp-fix.demo.clinicalgenome.org

Test steps:
1. Create a group, family, individual or case control evidence in GDM. 
1. Delete the evidence. 
1. Navigate back to the dashboard. In the "Recent History" panel, you should see the the creation and deletion record. For the delete record, it should show something like "\<your group name\> deleted; \<timestamp\>". Note down the timestamp.
1. Wait for a minute and stay at the dashboard page, so that the current time differs from the delete time.
1. Refresh the browser. You should see the delete timestamp stays the same and does not change over time. You can refer to the current time in the system time on your laptop, to help you observe the time change.
